samcli/local/docker/container.py

```diff
@@ -123,16 +126,33 @@ def create(self):
 
         _volumes = {}
 
+        # Default volume mount mode and message.
```

Comment:
minor: could be a better idea to refactor the whole `mount_mode` part to a `_get_mount_mode` method

Reply:
Do you have a good rubric for understanding when these would be "Z,ro,delegated" and when delegated would be absent? I don't know how to write a `_get_mount_mode` function unless I know how and why those mount options are applied in each situation used here.